RNTester/Podfile.lock

```diff
@@ -431,7 +431,7 @@ DEPENDENCIES:
   - Yoga (from `../ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://cdn.cocoapods.org/:
+  trunk:
```

Comment:
For someone completely unfamiliar with Cocoapods, is this something that should be checked in?

Reply:
As someone who _kinda_ knows it, I had to google this 😅 Can you try the steps shown here? https://willowtreeapps.com/ideas/how-to-fix-unexpected-podfile-lock-diffs-after-pod-install